### PR TITLE
Add "http.query.string" and "http.fragment.string" to DDTags

### PR DESCRIPTION
**Pull request by randomanderson**
Removes duplicate strings in HttpServerDecorator and HttpClientDecorator by replacing with a constant in DDTags

### CODE REVIEW

File: dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpClientDecorator.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import datadog.trace.api.DDTags;
 import io.opentracing.Span;
 import io.opentracing.tag.Tags;
+
```
Comment (labbati) [inline dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpClientDecorator.java]: We perform a check to make sure that all code complies to the google standard for java files. You can run this command and you code styles will be fixed in place automatically: `./gradlew googleJavaFormat.

Some more details and tips here: https://github.com/DataDog/dd-trace-java/blob/master/CONTRIBUTING.md#code-style

File: dd-java-agent/agent-tooling/src/test/groovy/datadog/trace/agent/decorator/HttpClientDecoratorTest.groovy [hunk 2]
```diff
@@ -58,8 +58,8 @@ class HttpClientDecoratorTest extends ClientDecoratorTest {
       1 * span.setTag(Tags.HTTP_URL.key, expectedUrl)
     }
     if (expectedUrl && tagQueryString) {
-      1 * span.setTag("http.query.string", expectedQuery)
-      1 * span.setTag("http.fragment.string", expectedFragment)
+      1 * span.setTag(DDTags.HTTP_QUERY, expectedQuery)
+      1 * span.setTag(DDTags.HTTP_FRAGMENT, expectedFragment)
```
Comment (labbati) [inline dd-java-agent/agent-tooling/src/test/groovy/datadog/trace/agent/decorator/HttpClientDecoratorTest.groovy]: This is sort of debatable so I will leave the choice to you. The point is that if by mistake one changes the value in the constant in the `DDTags` class then this test would pass. My preference here is to keep use the hard-coded value in tests. I know there might be different opinions here, so I leave the choice to you.

File: dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpClientDecorator.java (hunk 1 above)
Comment (randomanderson) [reply dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpClientDecorator.java]: Weird bug in the Save Actions plugin:

- If I manually run "Optimize Imports", it removes the blank line.
- If I save the file with other problems in the imports, it removes the blank line
- If I save the file with only the blank line as the issue, it doesn't do anything!

I'll remember to run googleJavaFormat going forward

File: dd-java-agent/agent-tooling/src/test/groovy/datadog/trace/agent/decorator/HttpClientDecoratorTest.groovy (hunk 2 above)
Comment (randomanderson) [reply dd-java-agent/agent-tooling/src/test/groovy/datadog/trace/agent/decorator/HttpClientDecoratorTest.groovy]: I'm going to disagree with you here.  Everywhere else in the file, the tests use constant versions including DDTags (https://github.com/DataDog/dd-trace-java/blob/ad822d81f974613c4e75772b153c06669c7034d5/dd-java-agent/agent-tooling/src/test/groovy/datadog/trace/agent/decorator/HttpClientDecoratorTest.groovy#L34)

Comment (labbati) [reply dd-java-agent/agent-tooling/src/test/groovy/datadog/trace/agent/decorator/HttpClientDecoratorTest.groovy]: Yep just noticed it and good point. Apart from this specific file what are your thoughts about this? (for future reference)

**Pull request merged**